huaweicloud/resource_huaweicloud_compute_instance_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/chnsz/golangsdk/openstack/common/tags"
 	"github.com/chnsz/golangsdk/openstack/compute/v2/servers"
+	"github.com/chnsz/golangsdk/openstack/ecs/v1/cloudservers"
```

Comment:
suggest to replace all v2/servers APIs with v1 APIs in the testing.

Reply:
done